spec/test_cbw_api.py

```diff
@@ -99,12 +98,48 @@ def test_update_server():
             assert response is True
 
     @staticmethod
-    def test_remote_accesses():
-        """Tests for method remote_accesses"""
+    def test_update_remote_access():
+        """Tests for update remote method"""
         client = CBWApi(API_URL, API_KEY, SECRET_KEY)
 
-        with vcr.use_cassette('spec/fixtures/vcr_cassettes/remote_accesses.yaml'):
-            response = client.remote_accesses()
-            assert isinstance(response, list) is True
-            for remote in response:
-                assert isinstance(remote, CBWRemoteAccess) is True
+        info = {"type": "CbwRam::RemoteAccess::Ssh::WithPassword",
+                "address": "10.10.10.228",
+                "port": "22",
+                "login": "loginssh",
+                "password": "passwordssh",
+                "key": "",
+                "node": "master"
+                }
+
+        with vcr.use_cassette('spec/fixtures/vcr_cassettes/update_remote_access.yaml'):
+            response = client.update_remote_access('4', info)
+
+            assert response is True
+
+        info = {"type": "CbwRam::RemoteAccess::Ssh::WithPassword",
```

Comment:
there are no difference with the previous one, remove.

Reply:
there is one in the address

Reply:
Ok :-1: